packages/components/sonar-project.properties

```diff
@@ -9,23 +9,37 @@ sonar.exclusions = \
   **/*.test.tsx, \
   **/*.spec.tsx, \
   \
+  # Storybook files
+  **/*.stories.tsx, \
+  \
   # Type definitions
   **/*.d.ts, \
   \
   # Source code to exclude (assets, styles, types)
+  # Source code to exclude (assets, styles, types, etc.)
   src/assets/**, \
   src/styles/**, \
   src/types/**, \
+  src/loggers/**, \
+  src/edge-runtime-compatible/**, \
```

Comment:
What is the argument for excluding these? They have tests already, although not 100% coverage.